core/Cargo.toml

```diff
@@ -61,17 +61,18 @@ solana-runtime = { path = "../runtime", version = "1.1.22" }
 solana-sdk = { path = "../sdk", version = "1.1.22" }
 solana-stake-program = { path = "../programs/stake", version = "1.1.22" }
 solana-storage-program = { path = "../programs/storage", version = "1.1.22" }
+solana-storage-bigtable = { path = "../storage-bigtable", version = "1.1.22" }
 solana-streamer = { path = "../streamer", version = "1.1.22" }
 solana-version = { path = "../version", version = "1.1.22" }
 solana-vote-program = { path = "../programs/vote", version = "1.1.22" }
 solana-vote-signer = { path = "../vote-signer", version = "1.1.22" }
 solana-sys-tuner = { path = "../sys-tuner", version = "1.1.22" }
 tempfile = "3.1.0"
 thiserror = "1.0"
-tokio = "0.1"
-tokio-codec = "0.1"
-tokio-fs = "0.1"
-tokio-io = "0.1"
+tokio = { version = "0.2.22", features = ["full"] }
+tokio_01 = { version = "0.1", package = "tokio" }
```

Comment:
that makes me queasy

Reply:
No doubt.   Until we boot the jsonrpc-* crates we're stuck with tokio 0.1 in the tree though -- they're using hyper 0.12.  Best we can we do now is port everything that doesn't touch jsonrpc-* to tokio 0.2